src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
+
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` if the operation's success can be determined by exceptions or other means. Returning a `Boolean` might not be necessary.